### PR TITLE
Add link to pod statistics in right navigation

### PR DESCRIPTION
**Pull request by SansPseudoFix**
Result:

link_for_stat

### CODE REVIEW

File: app/views/shared/_links.haml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 %li= link_to 'wiki', "https://wiki.diasporafoundation.org"
 %li= link_to t('layouts.application.whats_new'), changelog_url
 %li= link_to t('layouts.header.code') + " " + pod_version, "#{source_url}", {:title => t('layouts.application.source_package')}
+%li= link_to t('layouts.application.statistics_link'), statistics_path
```
Comment (svbergerem) [inline app/views/shared/_links.haml]: please use doublequotes

**Comment by goobertron** [comment]
Good idea. What would you think about using 'Network statistics' as the label, to make it more clear what these statistics are?

File: app/views/shared/_links.haml (hunk 1 above)
Comment (SansPseudoFix) [inline app/views/shared/_links.haml]: Okay. Do I do the same for other lines, or just mine?

**Comment by SansPseudoFix** [comment]
Yes, I didn't know which word to use, so I just used "Statistics". To be more specific, I can use "Pod statistics" maybe ("Network statistics" can be confused -> all diaspora* network).

**Comment by goobertron** [comment]
Oh, I'm sorry. Without checking the code, I assumed this  was a link to the network statistics page (http://the-federation.info/). If it is to the pod's own statistics page, then yes, 'Pod statistics' would be appropriate. I suppose this proves that 'Statistics' alone would be confusing!

**Comment by goobertron** [comment]
(And now I noticed that your PR has 'pod statistics' in its title - time for another eye test...)

**Comment by SansPseudoFix** [comment]
:D 
But you're right, "Statistics" only is confusing. I will change that for "Pod statistics".

File: app/views/shared/_links.haml (hunk 1 above)
Comment (svbergerem) [inline app/views/shared/_links.haml]: Just change the lines you already changed.

**Comment by SansPseudoFix** [comment]
Done.

**Comment by svbergerem** [comment]
Thank you.

**Pull request closed without merging**